WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailPasswordFragment.java

```diff
@@ -28,44 +15,32 @@
 import org.greenrobot.eventbus.Subscribe;
 import org.greenrobot.eventbus.ThreadMode;
 import org.wordpress.android.R;
-import org.wordpress.android.WordPress;
-import org.wordpress.android.fluxc.Dispatcher;
 import org.wordpress.android.fluxc.generated.AuthenticationActionBuilder;
 import org.wordpress.android.fluxc.store.AccountStore;
 import org.wordpress.android.fluxc.store.AccountStore.AuthenticatePayload;
 import org.wordpress.android.fluxc.store.AccountStore.OnAuthenticationChanged;
 import org.wordpress.android.util.AppLog;
 import org.wordpress.android.util.AppLog.T;
-import org.wordpress.android.util.EditTextUtils;
 import org.wordpress.android.util.NetworkUtils;
 import org.wordpress.android.util.ToastUtils;
+import org.wordpress.android.widgets.WPLoginInputRow;
+import org.wordpress.android.widgets.WPLoginInputRow.OnEditorCommitListener;
 
-import javax.inject.Inject;
-
-public class LoginEmailPasswordFragment extends Fragment implements TextWatcher {
-    private static final String KEY_IN_PROGRESS = "KEY_IN_PROGRESS";
+public class LoginEmailPasswordFragment extends LoginBaseFormFragment implements TextWatcher, OnEditorCommitListener {
     private static final String KEY_REQUESTED_PASSWORD = "KEY_REQUESTED_PASSWORD";
 
     private static final String ARG_EMAIL_ADDRESS = "ARG_EMAIL_ADDRESS";
     private static final String ARG_PASSWORD = "ARG_PASSWORD";
 
```

Comment:
I see the same constant name and value used across multiple files.  In the screenshot below from a quick search in Android Studio shows a few constants are named the same, but have a different value (e.g. `ARG_EMAIL_ADDRESS` can be `"ARG_EMAIL_ADDRESS"` or `"arg_email_address"`) or modifier (e.g. `ARG_PASSWORD` can be `private` or `public`).

<img width="826" alt="constants" src="https://user-images.githubusercontent.com/3827611/28465751-24486f2c-6de7-11e7-96c7-cdb56a2dcd25.png">

They may be used locally for now, but it may be hard to troubleshoot if they are ever used outside of their class in the future.